src/features/unified-share-modal/SharedLinkSection.js

```diff
@@ -197,6 +194,24 @@ class SharedLinkSection extends React.Component<Props, State> {
                     onCopyError();
                 });
         }
+
+        // if ESL ftux tooltip is showing on initial mount, we call onShow
+        const allowedPermissionLevels = this.getAllowedPermissionLevels(
```

Comment:
why do we need to pass all these as parameters?
can we directly access these props in the method `getAllowedPermissionLevels`?

Reply:
Yes good catch - will fix.